scripts/python/add_users_everyone_github_team.py

```diff
@@ -103,7 +80,34 @@ def run():
         print_stack_trace(message)
 
 
-print("Start")
-run()
-print("Finished")
-sys.exit(0)
+def main():
+    if len(sys.argv) == 2:
+        # Get the GH Action token
+        oauth_token = sys.argv[1]
+    else:
+        raise ValueError("Missing a script input parameter")
```

Comment:
only functional change was changing this to a `ValueError` from the `sys.exit(1)`. I only did this as the testing seemed more natural, but not 100% sure if there's a preference.

Reply:
No problem, as long as it bugs out the app at that point then it is okay, assume it does that?

Reply:
Believe it stops the execution (ref: https://docs.python.org/3/tutorial/errors.html#raising-exceptions)

So we're still failing fast 😁 💨 🔥

Reply:
Nice